src/event.js

```diff
@@ -573,7 +574,7 @@ jQuery.Event.prototype = {
 
 		this.isPropagationStopped = returnTrue;
 
-		if ( e ) {
+		if ( e && this.executeOriginalPropagation ) {
```

Comment:
I could have used `isSimulated` but choose not to, since such use wasn't intended for that property